### DISCUSSION TITLE
Sh1mmer Files

### DISCUSSION THREAD

**Discussion by ehunt34** [3kh0/ext-remover#410] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Yes, I'm back. As much as I don't want to admit it, I need some help. Since CoolElectronics can't make up his damn mind about whether or not to keep exploits up, I now have to compile all of his shims myself. My plan was to recompile all of them then keep them on my NeoChromium repository. However, I need to get the Original RMA Shim files in order to make them. I only have 14 of the 24 boards listed as supported, those being clapper, coral, enguarde, glimmer, grunt, hana, octopus, orco, pyro, reks, sentry, stout, tidus, and ultima. I am missing brask, brya, dedede, hatch, jacuzzi, kukui, nami, strongbad, volteer, and zork. I need either the RMA Shim files or the original pre-built sh1mmer files, especially for jacuzzi, as that is the board I have. Without that, I can't make a proof of concept for my own exploit. If you have one of those, please send it to me through Google Drive and send me the link privately. (make sure r/k12sysadmin can't get them). In return I will give you credit and give my pre-built shims to this repository.

**Comment by ehunt34** [comment]
For contact, I don't have Discord, but I have a Steam under ehunt1011 and a Reddit under ehunt34.

**Reply by Aka-but-nice** [reply]
@ehunt34 Just make a discord, we can talk, but ask your mom can you download discord on her phone

**Reply by ehunt34** [reply]
I'm trying to be reasonable yet you continue to be problematic, blocked.

**Comment by 3kh0** [comment, marked as answer]
lol you can find them xd, it is in the lauch youtube video, check the desc

**Reply by ehunt34** [reply]
Thanks. I'm sorry for my previous actions. I'll upload them and give this repository credit.

**Reply by rwg100** [reply]
ngl this is like in every movie ever when the villian joins the other side and they defeat some final boss together

**Reply by Aka-but-nice** [reply]
fr

**Reply by r58Playz** [reply]
I honestly don't know how you managed to find this before the original repo

**Comment by ProgrammerIn-wonderland** [comment]
I find it funny you wanted to give 3kh0 credit for sh1mmer when he had nothing to do with it, instead of the original team ofc

**Comment by Aka-but-nice** [comment]
> Yes, I'm back. As much as I don't want to admit it, I need some help. Since CoolElectronics can't make up his damn mind about whether or not to keep exploits up, I now have to compile all of his shims myself. My plan was to recompile all of them then keep them on my NeoChromium repository. However, I need to get the Original RMA Shim files in order to make them. I only have 14 of the 24 boards listed as supported, those being clapper, coral, enguarde, glimmer, grunt, hana, octopus, orco, pyro, reks, sentry, stout, tidus, and ultima. I am missing brask, brya, dedede, hatch, jacuzzi, kukui, nami, strongbad, volteer, and zork. I need either the RMA Shim files or the original pre-built sh1mmer files, especially for jacuzzi, as that is the board I have. Without that, I can't make a proof of concept for my own exploit. If you have one of those, please send it to me through Google Drive and send me the link privately. (make sure r/k12sysadmin can't get them). In return I will give you credit and give my pre-built shims to this repository.

I really do not understand why we're even beefing with you, you're pretty much any other civilized person using the internet.

**Reply by r58Playz** [reply]
@Aka-unblock 

Maybe because ehunt34 is giving credit to this knockoff repo instead of the real team over at Mercury Workshop.

**Comment by ProgrammerIn-wonderland** [comment]
Also the taking down the shims part was my decision, not cool electronics, if you want the shims so bad go use Julia's mirror lmao

**Reply by Capsiscapin** [reply]
whats julia's mirror
